### PR TITLE
Add STDCI

### PR DESCRIPTION
**Pull request by gbenhaim**
Signed-off-by: gbenhaim <galbh2@gmail.com>

### CODE REVIEW

File: automation/check-patch.packages [hunk 1]
```diff
@@ -0,0 +1,2 @@
+docker
+golang-1.9.4
```
Comment (SchSeba) [inline automation/check-patch.packages]: we need to use golang > 1.9

**Comment by gbenhaim** [comment]
@SchSeba I removed the go installation.

**Comment by SchSeba** [comment]
@gbenhaim can we configure it just like kubevirt?
```
git
rsync
docker
make
```

**Comment by phoracek** [comment]
ci test please

File: automation/check-merged.environment.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+- name: REGISTRY_PASSWORD
+  valueFrom:
+    secretKeyRef:
+      name: quay
+      key: kubevirt_bot
```
Comment (phoracek) [inline automation/check-merged.environment.yaml]: Once there is a bot created for CI, would you please add just `docker login` to check-merged.sh? We will add the rest then.

Comment (gbenhaim) [reply automation/check-merged.environment.yaml]: Sure. Can we do it in a different PR?

**Comment by gbenhaim** [comment]
@SchSeba I've updated to packages list.

File: automation/check-patch.packages [hunk 3]
```diff
@@ -0,0 +1,4 @@
+git
```
Comment (phoracek) [inline automation/check-patch.packages]: Do we need all of these now? Please drop unused dependencies.

File: automation/check-merged.environment.yaml (hunk 2 above)
Comment (phoracek) [reply automation/check-merged.environment.yaml]: Fair enough

File: automation/check-patch.packages (hunk 3 above)
Comment (gbenhaim) [reply automation/check-patch.packages]: It was requested by @SchSeba

Comment (SchSeba) [reply automation/check-patch.packages]: @phoracek I think the only one we can drop is git no?

Comment (phoracek) [reply automation/check-patch.packages]: Also I don't see rsync and make used.

Comment (SchSeba) [reply automation/check-patch.packages]: Ok lets try.
This was copy from the kubevirt repo.

**Comment by gbenhaim** [comment]
@phoracek I've managed to squeeze the login to quay to this PR.
I think that we can merge it.

**Comment by phoracek** [comment]
@gbenhaim that's perfect! Thanks a lot.

**Pull request merged**